testnet/stacks-node/src/tests/signer/v0.rs

```diff
@@ -11237,20 +11250,33 @@ fn allow_reorg_within_first_proposal_burn_block_timing_secs() {
         send_amt,
     );
     let tx = submit_tx(&http_origin, &transfer_tx);
-    info!("Submitted tx {tx} in attempt to mine block N+2");
+    info!("Submitted tx {tx} in attempt to mine block N+2'");
 
     wait_for(30, || {
         Ok(
             blocks_mined1.load(Ordering::SeqCst) > blocks_processed_before_1
                 && test_observer::get_mined_nakamoto_blocks().len() > mined_before,
         )
     })
-    .expect("Timed out waiting for Miner 1 to Mine Block N+2");
+    .expect("Timed out waiting for Miner 1 to Mine Block N+2'");
 
     let blocks = test_observer::get_mined_nakamoto_blocks();
     let last_block = blocks.last().expect("No blocks mined");
     assert_eq!(last_block.stacks_height, block_n_height + 2);
 
+    info!("------------------------- Miner 1 Mines N+4 in Next Tenure -------------------------");
+
+    next_block_and_process_new_stacks_block(
+        &mut signer_test.running_nodes.btc_regtest_controller,
+        30,
+        &signer_test.running_nodes.coord_channel,
+    )
+    .expect("Timed out waiting for Miner 1 to Mine Block N+4");
+
+    let blocks = test_observer::get_mined_nakamoto_blocks();
+    let last_block = blocks.last().expect("No blocks mined");
+    assert_eq!(last_block.stacks_height, block_n_height + 4);
```

Comment:
To be clear, block N+1 builds atop miner 2, right?  Not miner 1's N+2'?

Reply:
This is miner 1 building block N+4 atop miner 2's block N+3, reorging its own N+1' and N+2' from the previous tenure. Maybe I can add some more checks in here, to make the parent block explicit.

Reply:
Ok, I updated the test with explicit parent block checks.